themes/Suite7/css/style.css

```diff
@@ -1413,6 +1413,43 @@ width:350px;
     margin-bottom: 0;
 }
 
+
+#scheduler .schedulerDiv table tr.schedulerAttendeeRow td.schedulerSlotCellStartTime.free {
+    border-left: 4px solid  #ff0000;
+}
+
+#scheduler .schedulerDiv table tr.schedulerAttendeeRow td.schedulerSlotCellStartTime.busy {
+    border-left: 4px solid  #ff0000;
+}
+
+#scheduler .schedulerDiv table tr.schedulerAttendeeRow td.schedulerSlotCellStartTime {
+    border-left: 4px solid #75AF4F /*!important*/;
```

Comment:
This was due to me copying the code from SuiteP. As I wanted to be able to make the colors in the scheduling bar customisable. I have amended this file and the SuiteR version.